src/helpers/stellarCli.ts

```diff
@@ -0,0 +1,2 @@
+// Stub file: the real file is generated in build time by `yarn stellar-cli:build`.
+export const latestVersion = "0.0.0";
```

Comment:
Should we still commit latest version? If you build locally, there'll be unstated git changes

Reply:
I don't think we should. Running local dev doesn't update this file, it's unlikely that you'll build the final site (we do something similar with the manual), but if you do, having 0.0.0 is a great way of knowing that it didn't work properly.